docs/conf/portland/2018/cfp.rst

```diff
@@ -0,0 +1,105 @@
+:template: 2018/base.html
+
+Call for proposals
```

Comment:
general question - can we move away from mid-sentence line breaks? IIRC it doesn't make a difference in publications but it helps to have a one-sentence-per-line with appropriate soft tab enabled in text editor.

Reply:
+1 makes it a lot easier to review, though I still like line-break on punctuation over sentence, because often commas seperate ideas :) It falls down on lists though, so _shrug_.

Reply:
I have a preference for short lines, esp on a small screen, but can go with the majority.

Reply:
I also prefer short lines for diffs, oddly enough.